assets/js/modules/analytics/components/module/ModuleAcquisitionChannelsWidget/AcquisitionChannelsTable.js

```diff
@@ -0,0 +1,108 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { numFmt } from '../../../../../util';
+import MiniChart from '../../../../../components/MiniChart';
+import ReportTable from '../../../../../components/ReportTable';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+const { useSelect } = Data;
+
+export default function AcquisitionChannelsTable( { report } ) {
+	const dateRangeNumberOfDays = useSelect( ( select ) => select( CORE_USER ).getDateRangeNumberOfDays() );
+
+	const totalUsers = report[ 0 ].data.totals[ 0 ].values[ 1 ];
+	let iterator = -1; // We pre-increment, hence starting at -1.
+
+	const tableColumns = [
+		{
+			title: __( 'Channel', 'google-site-kit' ),
+			tooltip: __( 'Channel refers to where your traffic originated from', 'google-site-kit' ),
+			primary: true,
+			Component: ( { row } ) => row.dimensions[ 0 ],
+		},
+		{
+			title: __( 'Users', 'google-site-kit' ),
+			tooltip: __( 'Number of users that originated from that traffic', 'google-site-kit' ),
+			field: 'metrics.0.values.0',
+			Component: ( { fieldValue } ) => numFmt( fieldValue, { style: 'decimal' } ),
+		},
+		{
+			title: __( 'New Users', 'google-site-kit' ),
+			tooltip: sprintf(
+				/* translators: %s: number of days */
+				_n( 'Number of new users to visit your page over last %s day', 'Number of new users to visit your page over last %s days', dateRangeNumberOfDays, 'google-site-kit', ),
+				dateRangeNumberOfDays,
+			),
+			field: 'metrics.0.values.1',
+			Component: ( { fieldValue } ) => numFmt( fieldValue, { style: 'decimal' } ),
+		},
+		{
+			title: __( 'Sessions', 'google-site-kit' ),
+			tooltip: sprintf(
+				/* translators: %s: number of days */
+				_n( 'Number of sessions users had on your website over last %s day', 'Number of sessions users had on your website over last %s days', dateRangeNumberOfDays, 'google-site-kit', ),
+				dateRangeNumberOfDays,
+			),
+			field: 'metrics.0.values.2',
+			Component: ( { fieldValue } ) => numFmt( fieldValue, { style: 'decimal' } ),
+		},
+		{
+			title: __( 'Percentage', 'google-site-kit' ),
+			tooltip: __( 'Percentage of sessions', 'google-site-kit' ),
+			field: 'metrics.0.values.1',
+			Component: ( { fieldValue } ) => {
+				const change = fieldValue / totalUsers;
+				iterator += 1;
+				return (
+					<Fragment key={ 'minichart-analytics-top-as-' + iterator }>
+						<div className="googlesitekit-table__body-item-chart-wrap">
+							{ numFmt( change, '%' ) }
+							<MiniChart change={ change } index={ iterator } />
+						</div>
+					</Fragment>
```

Comment:
`<Fragment>` seems to be redundant here, let's remove it then:

```suggestion
					<div key={ 'minichart-analytics-top-as-' + iterator } className="googlesitekit-table__body-item-chart-wrap">
						{ numFmt( change, '%' ) }
						<MiniChart change={ change } index={ iterator } />
					</div>
```